openxr/examples/vulkan.rs

```diff
@@ -440,14 +488,17 @@ fn main() {
                         usage_flags: xr::SwapchainUsageFlags::COLOR_ATTACHMENT
                             | xr::SwapchainUsageFlags::SAMPLED,
                         format: COLOR_FORMAT.as_raw() as _,
-                        sample_count: 1,
+                        sample_count: views[0].recommended_swapchain_sample_count,
```

Comment:
The Vulkan graphics pipeline we create is not set up for multisampling, so we should either leave this hardcoded to 1 or pursue a proper multisampling setup. I don't think there's any compatibility hazard for leaving it at 1.

Reply:
Aha, that makes sense. I'll revert this and add documentation to this effect.